include/mgp.hpp

```diff
@@ -19,7 +19,9 @@
 #include <vector>
 
 #include "_mgp.hpp"
+#include "mg_exceptions.hpp"
 #include "mg_procedure.h"
+#include "utils/fnv.hpp"
```

Comment:
There is an issue that when running the MAGE setup script this need to be changed to 
`#include "../src/utils/fnv.hpp"`
There should be a more elegant way to fix this but I am not well versed in Cmake

Reply:
fnv.hpp was copy pasted inside mgp::util so MAGE doesn't have any dependacies.